app/views/shared/forms/_next_action_buttons.html.erb

```diff
@@ -0,0 +1,12 @@
+<%
+  show_draft = false unless local_assigns.key?(:show_draft)
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form.submit t('generic.continue'), id: continue_id, name: 'continue-button', class: 'govuk-button form-button' %>
```

Comment:
If you name the submit buttons `continue_button` and `draft_button` the code elsewhere becomes a lot neater because these labels can then be represented via symbols. 

Hyphens are more typical of HTML markup, but underscores already litter the markup (its a consequence of using Rails). Therefore, I'd argue that the cost of using more HTML friendly hyphens is outweighed by the extra work this causes in the Ruby space.